src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/SolutionExplorer/PackageManagerUICommandHandler.cs

```diff
@@ -314,10 +314,11 @@ private async Task<System.Windows.Window> CreateNewWindowFrameAsync(string proje
 
             var vsWindowSearchHostfactory = await _asyncServiceProvider.GetServiceAsync(typeof(vsShellInterop.SVsWindowSearchHostFactory)) as vsShellInterop.IVsWindowSearchHostFactory;
             var vsShell = await _asyncServiceProvider.GetServiceAsync(typeof(vsShellInterop.SVsShell)) as vsShellInterop.IVsShell4;
-			// TODO: rob to fix assembly loading problem.
-            //var control = new PackageManagerControl(model, Settings.Value, vsWindowSearchHostfactory, vsShell, OutputConsoleLogger.Value);
 
-            //var windowPane = new PackageManagerWindowPane(control);
+            // TODO: rob to fix assembly loading problem.
```

Comment:
let's assume you fixed the assembly loading problem...and remove this TODO for now.

Reply:
(since you aren't seeing the problem on your machine)